docs/demos/coast-to-coast-q4-2023.md

```diff
@@ -0,0 +1,950 @@
+# Coast-to-coast Demo for Q4 2023
+
+This coast-to-coast demo highlights some of the new features introduced to OLMv1 in Q4 of 2023.
+
+New Features:
+- `Catalog` polling
+- Version range support on the `spec.version` field for `Operator` resources
+- Semver upgrade constraint policy
+- `BundleDeployment` health status
+
+This document will go through an example that highlights each of these new features. Each step will be documented as if you are following along and will be done sequentially. This document will also use some values specific to this example and in your own projects should be replaced to use a different value.
+
+## Prerequisites
+- A running Kubernetes cluster where you have admin privileges
+- `operator-sdk` installed
+    - Installation instructions can be found at https://sdk.operatorframework.io/docs/installation/
+- `kubectl` installed
+    - Installation instructions can be found at https://kubernetes.io/docs/tasks/tools/
+- A container runtime of your choice, in this demo we will be using `docker`
+- Install `operator-controller` vX.Y.Z 
+    - Installation instructions can be found at https://github.com/operator-framework/operator-controller/releases/tag/vX.Y.Z
```

Comment:
I believe the only thing left is to replace the `vX.Y.Z` references here with the next release of operator-controller and then I can mark it as "ready for review" (although I feel this has been fairly thoroughly reviewed and should be pretty much ready to merge)

Reply:
Updated to v0.8.0 in https://github.com/operator-framework/operator-controller/pull/533/commits/e4c53aa15e95ebdc0144e6fc968396a195bacb34